environment.yml

```diff
@@ -23,12 +23,13 @@ dependencies:
   - distributed
   - ecmwf-api-client
   - eofs
-  - esmpy >=8.6.0  # github.com/SciTools-incubator/iris-esmf-regrid/pull/342
+  - esmpy !=8.1.0,<8.6.0  # https://github.com/SciTools-incubator/iris-esmf-regrid/pull/342#issuecomment-2092921514
```

Comment:
Is this upper pin needed? If NCL already pins ESMPy, we don't need to do it, right?

Reply:
yes, absolutely! I was a bit too cautious, will remove now

Reply:
Core: https://github.com/ESMValGroup/ESMValCore/blob/99c6a7be1e995f8b4fcb61f3a44bd46aa828c397/environment.yml#L15

Reply:
The pull request where you introduce the pin in ESMValCore https://github.com/ESMValGroup/ESMValCore/pull/2542 does not explain why it is pinned either. It seems unlikely that such an old version of esmpy will be picked up.

Reply:
Found it! Holy cow, a nasty one indeed https://github.com/ESMValGroup/ESMValCore/pull/1310

Reply:
I removed it, that PR is from 3+ years ago, if Mamba finds a solution with that version, it means we're in serious trouble elsewhere not just esmpy

Reply:
Thanks!